gluten-substrait/src/main/scala/org/apache/gluten/execution/WholeStageTransformer.scala

```diff
@@ -377,26 +378,28 @@ case class WholeStageTransformer(child: SparkPlan, materializeInput: Boolean = f
       val allScanSplitInfos =
         getSplitInfosFromPartitions(basicScanExecTransformers, allScanPartitions)
       if (GlutenConfig.getConf.enableHdfsViewfs) {
+        val viewfsToHdfsCache: mutable.Map[String, String] = mutable.Map.empty
```

Comment:
Would this cache take too much memory?

Reply:
No, the size of this map is at most equal to the total number of partitions of these tables.